splunk_connect_for_snmp/inventory/loader.py

```diff
@@ -156,6 +105,64 @@ def configure_ui_database(mongo_client):
         profiles_ui_collection.drop()
 
 
+def add_groups_and_profiles_from_config_runtime(
+    groups_ui_collection, profiles_ui_collection
+):
+    groups = {}
+    all_profiles = {}
+    try:
+        with open(CONFIG_PATH, encoding="utf-8") as file:
+            config_runtime = yaml.safe_load(file)
+            if "groups" in config_runtime:
+                groups = config_runtime.get("groups", {})
+
+            if "profiles" in config_runtime:
+                profiles = config_runtime.get("profiles", {})
+                logger.info(
+                    f"loading {len(profiles.keys())} profiles from runtime profiles config"
+                )
+                for key, profile in profiles.items():
+                    all_profiles[key] = profile
+    except FileNotFoundError:
+        logger.info(f"File: {CONFIG_PATH} not found")
+    groups_list = [{key: value} for key, value in groups.items()]
+    if groups_list:
+        groups_ui_collection.insert_many(groups_list)
+    profiles_list = [{key: value} for key, value in all_profiles.items()]
+    if profiles_list:
+        profiles_ui_collection.insert_many(profiles_list)
+
+
+def assign_inventory_values(inventory_ui_collection):
+    with open(INVENTORY_PATH, encoding="utf-8") as csv_file:
+        ir_reader = DictReader(csv_file)
+        all_inventory_lines = []
+        for inventory_line in ir_reader:
+            for key in INVENTORY_KEYS_TRANSFORM.keys():
+                if key in inventory_line:
+                    new_key = INVENTORY_KEYS_TRANSFORM[key]
+                    inventory_line[new_key] = inventory_line.pop(key)
+
+            for field in BOOLEAN_INVENTORY_FIELDS:
+                inventory_line[field] = human_bool(inventory_line[field], False)
+
+            port = (
+                int(inventory_line.get("port", 161))
+                if inventory_line.get("port", 161)
+                else 161
+            )
+            walk_interval = (
+                int(inventory_line["walk_interval"])
```

Comment:
I think it is like that because this different db collection that is keeping data from/for UI the `mongo_client.sc4snmp.inventory_ui`.  It is later merged with the general one. The `InventoryRecord` is targeting different db `mongo_client.sc4snmp.inventory`.